translations/messages.en.xlf

```diff
@@ -2655,6 +2655,29 @@
         <target>Edit Permissions</target>
       </segment>
     </unit>
+    <unit id="a_COwtV" name="image.button_upload_options">
+      <segment>
+        <source>image.button_upload_options</source>
+        <target>Upload options</target>
+      </segment>
+    </unit>
+    <unit id="OYUVA5." name="general.label.search">
+      <segment>
+        <source>general.label.search</source>
+        <target>Search</target>
+      </segment>
+    </unit>
+    <unit id="hKmDb7q" name="Share secure preview link">
+      <segment>
```

Comment:
@I-Valchev I can't really figure out why it thinks I Added this translation. But its still the same as on master, so I guess it's ok?